### ISSUE TITLE
abnormal,increased,decreased brain N-acetyl-L-aspartate level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3586]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

A. abnormal brain N-acetyl-L-aspartate level
B. increased brain N-acetyl-L-aspartate level
C. decreased brain N-acetyl-L-aspartate level

>2. Synonyms (e.g., absent spleen)

A. aberrant brain N-acetyl-L-aspartate level; aberrant brain NAA level; abnormal brain NAA level
B. increased brain NAA level
C. reduced brain N-acetyl-L-aspartate level; reduced brain NAA level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

A. an anomaly in the amount of N-acetyl-L-aspartate, the substrate of aspartoacylase, present in brain tissue
B. an increase in the amount of N-acetyl-L-aspartate, the substrate of aspartoacylase, present in brain tissue
C. a decrease in the amount of N-acetyl-L-aspartate, the substrate of aspartoacylase, present in brain tissue

PMID:10820213; HP:0032273

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

A. abnormal brain morphology (MP:0002152)
B. abnormal brain N-acetyl-L-aspartate level (new term)
C. abnormal brain N-acetyl-L-aspartate level (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803

**Comment by sbello** [comment]
used the CHEBI term name N-acetyl-L-aspartic acid in the term names
abnormal - MP:0021142
increased - MP:0021143
decreased - MP:0021144

also added MP:0021145 abnormal amino acid derivative level to group these types of terms
